### ISSUE TITLE
Wordweb : while pressing up or down arrow to read next or previous meaning of the word typed NVDA incorrectly announces the current definition edit multiline read only the definition gaining focus

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4315]
**Reported by blindbhavya on 2014-07-19 08:12**
To understand and reproduce this issue kindly download and install Wordweb, which is a freeware.
When in the place where all the meanings of the currently typed word are displayed, and user presses up or down arrow to read the next or previous meaning, NVDA in definition, NVDA incorrectly announces current definition edit multiline read only and then the newly focused definition.
Steps to reproduce
Download, install and launch Wordweb.
Type any sensible English word and press Tab twice to get your focus in the place where meanings of the current word are displayed.
Ensure that the word you have typed has two or more definitions.
When tab is pressed twice and NVDA has finished reading the first definition (the definition should be of only one line), press the down arrow to read the next definition and hear the announcement.
I have described above what the result should be.
Ideally, NVDA should only read the newly focused definition.

**Comment by leonardder** [comment]
cc @bhavyashah , do you still consider this request important?

**Comment by bhavyashah** [comment]
@leonardder As critical I am of my 12-year-old self, I do think this issue is still quite important. Let me know if any further information pertaining to this issue is required to accordingly prioritize thicket.
P.S. Still chuckling at myself for using phrases like "Joseph Sir" in https://github.com/nvaccess/nvda/issues/4315#issuecomment-155322372 and "Mr. James Teh" in https://github.com/nvaccess/nvda/issues/4315#issuecomment-155322364. :D

**Comment by Adriani90** [comment]
@bhavyashah could you please test with the last Version of Wordweb and Report last changes? Is the issue still occuring?

**Comment by bhavyashah** [comment]
@Adriani90 @leonardder Thanks for the follow-ups! I cannot reproduce this issue with WordWeb Pro version 10.2.0.0 and NVDA 2023.2. Feel free to close this ticket.

**Comment by Adriani90** [comment]
Closing as works for me.
